### ISSUE TITLE
ccleaner.sls breaks winrepo.genrepo

### ISSUE THREAD

**Issue by TheBigBear** [saltstack/salt-winrepo#155]
when running winrepo.genrepo it complains about blocks in ccleaner.sls

**Comment by UtahDave** [comment]
can you provide cli output on that?

**Comment by TheBigBear** [comment]
Hi,

Sorry I over-reacted. I had missed a missing “:”. Have fixed it and submitted PR.

Also found similar but milder winrepo.genrepo syntax issues of misaligned blocks in other files. have created PRs for them. All should be well.

**Comment by TheBigBear** [comment]
Ok, sure, sorry I didn;'t include it. It seemed obvious to e that it was my inexperience with python and keeping things nealty lined up in blocks.

```
salt-run winrepo.genrepo
Failed to compile /srv/salt/win/repo/salt-winrepo.git/ccleaner.sls: while parsing a block mapping
  in "<string>", line 3, column 5:
        installer: 'http://download.piri ... 
        ^
expected <block end>, but found '<block mapping start>'
  in "<string>", line 4, column 6:
         full_name: 'CCleaner 5.02'
         ^
```
My PR above fixes this and the winrepo.genrepo complaints go away so I think it fixes #156
